doc/data/messages/t/too-few-function-args/bad.py

```diff
@@ -0,0 +1 @@
+isinstance("foo")  # [too-few-function-args]
```

Comment:
Something minor for next time: in the pylint docs we tend to avoid "foo" and "bar" because junior programmers are often the target audience, and if they don't know the convention/backstory it has the potential to be distracting.

Also, `"foo"` is a literal just like `1` -- I was suggesting to change it for the reason that if we introduce a check later like `literal-isinstance` (in the spirit of `using-constant-test`), then we'll have to adjust this example. But let's not change it now, I'm not sure it's worth fleshing out this example more.